6.7.6-community/run.sh

```diff
@@ -14,14 +14,21 @@ fi
 
 declare -a sq_opts
 
+# Update UID:GID for sonarqube user.
+usermod -u $PUID sonarqube && \
+groupmod -g $PGID sonarqube && \
+chown -R -h $PUID:$PGID $SONARQUBE_HOME && \
+usermod -g $PUID sonarqube
```

Comment:
`usermod -g` changes the group; Should be `$PGID` ?
Same suggestion goes for `7.6-community/run.sh` at line 21

```suggestion
usermod -g $PGID sonarqube
```